llvm/test/CodeGen/AArch64/dllexport.ll

```diff
@@ -106,7 +106,7 @@ define weak_odr dllexport void @l() {
 ; CHECK-MSVC-EC: .ascii "  /EXPORT:o,DATA"
 ; CHECK-MSVC-EC: .ascii "  /EXPORT:p,DATA"
 ; CHECK-MSVC-EC: .ascii "  /EXPORT:q,DATA"
-; CHECK-MSVC-EC: .ascii "  /EXPORT:r"
-; CHECK-MSVC-EC: .ascii "  /EXPORT:s"
-; CHECK-MSVC-EC: .ascii "  /EXPORT:t"
-; CHECK-MSVC-EC: .ascii "  /EXPORT:u"
+; CHECK-MSVC-EC: .ascii "  /EXPORT:#r,EXPORTAS,r"
```

Comment:
Does this work with the MSVC linker?

Reply:
Yes, it works, I rechecked to be sure. Thanks!